.reek

```diff
@@ -45,6 +45,7 @@ FeatureEnvy:
     - Utf8Sanitizer#event_attributes
     - Utf8Sanitizer#remote_ip
     - Idv::Proofer#validate_vendors
+    - PersonalKeyGenerator#create_legacy_recovery_code
```

Comment:
Ignoring this here. This method will be deleted in the PR that stops reading from these columns